--- lockscreen-manager.c	2020-09-13 14:09:19.678181566 -0500
+++ lockscreen-manager2.c	2020-09-13 14:11:50.067686831 -0500
@@ -72,9 +72,6 @@
   PhoshShell *shell = phosh_shell_get_default ();
   PhoshMonitorManager *monitor_manager = phosh_shell_get_monitor_manager (shell);
 
-  phosh_shell_rotate_display (shell, priv->rotation);
-  priv->rotation = 0;
-
   g_return_if_fail (PHOSH_IS_LOCKSCREEN (lockscreen));
   g_return_if_fail (lockscreen == PHOSH_LOCKSCREEN (priv->lockscreen));
 
@@ -161,10 +158,6 @@
   primary_monitor = phosh_shell_get_primary_monitor (shell);
   g_return_if_fail (primary_monitor);
 
-  /* Undo any rotation so the keypad becomes usable */
-  priv->rotation = phosh_shell_get_rotation (shell);
-  phosh_shell_rotate_display (shell, 0);
-
   /* Listen for monitor changes */
   g_signal_connect_object (monitor_manager, "monitor-added",
                            G_CALLBACK (on_monitor_added),
